### ISSUE TITLE
Dasherize Data Attributes

### ISSUE THREAD

**Issue by jamesmartin** [jamesmartin/inline_svg#51]
Reported in #50. We should "dasherize" the names of data attributes:

```ruby
# Default:
<%= link_to 'a', '#', data: { tooltip_id: 'id', tooltip_placement: 'right' } %>
<a data-tooltip-id="id" data-tooltip-placement="right" href="#">a</a>

# InlineSvg:
<%= inline_svg 'caret-up.svg', data: { tooltip_id: 'id', tooltip_placement: 'right' } %>
<svg xmlns="http://www.w3.org/2000/svg" width="34" height="34" viewBox="0 0 34 34" data-tooltip_id="id" data-tooltip_placement="right">
  <path d="M17 4.541l16.04 23.919h-32.08z"></path>
</svg>
```

/cc @ollie

**Comment by jamesmartin** [comment]
Released in [v0.11.1](https://github.com/jamesmartin/inline_svg/releases/tag/v0.11.1).

**Comment by ollie** [comment]
Thanks!
